frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -478,6 +484,11 @@ public int getNumberOfGpu() {
         return getIntProperty(TS_NUMBER_OF_GPU, 0);
     }
 
+    public boolean getModelControlMode() {
+        String mode = getProperty(MODEL_CONTROL_MODE, "none");
+        return mode.equals("explicit") ? true : false;
+    }
+
```

Comment:
As I commented before, it is not efficient to compare every time. it should be only time job, return true/false directly starting from 2nd time

Reply:
I mean "mode.equals("explicit")" is not efficient.

Reply:
It is only being compared one time now, like you mentioned that its not efficient every time. Are you saying to remove the string comparison all together? Currently it is only being compared once instead of during every check as before.